File: IOS/DTOs/Enums/ContactType.swift
import Foundation

public enum ContactType: String, Codable, CaseIterable, Hashable, Equatable, Sendable {
    case phone
    case email
    case telegram
    case instagram
}

public enum PhoneTypes: String, Codable, CaseIterable, Hashable, Equatable, Sendable {
    case whatsapp
    case message
    case call
}

File: android-legacy/shared/src/main/kotlin/shared/dto/enums/ContactType.kt
package shared.dto.enums

import kotlinx.serialization.SerialName

enum class ContactType {
    @SerialName("phone")
    PHONE,

    @SerialName("email")
    EMAIL,

    @SerialName("telegram")
    TELEGRAM,

    @SerialName("instagram")
    INSTAGRAM,
}

enum class PhoneTypes {
    @SerialName("whatsapp")
    WHATSAPP,

    @SerialName("message")
    MESSAGE,

    @SerialName("call")
    CALL,
}